README.md

```diff
@@ -188,10 +188,13 @@ the plugin above:
 
 ```swift
 dependencies: [
-    .package(name: "SwiftProtobuf", url: "https://github.com/apple/swift-protobuf.git", from: "1.6.0"),
+    .package(url: "https://github.com/apple/swift-protobuf.git", from: "1.6.0"),
```

Comment:
So this is undoing what you sent in in #976, so I guess different versions of swift want different things?

Reply:
Yes, that's it in a nutshell. The manifest format for Package.swift continues to evolve, and this appears to be the "current" (swift 5.6/5.7) iteration. The older format does still work, and Xcode presents you with deprecation warnings in that case - so if you'd prefer to keep it "as is", I'd very much understand. When it was a deprecation warning, I figured it might be worth updating in the README though.

Reply:
@tbkka should we land this now or wait for 5.7 to GM before updating things?